CalibTracker/SiStripChannelGain/python/ntuple_cff.py

```diff
@@ -12,14 +23,53 @@
                                      'keep *_shallowTracks_tracketa_*',
                                      'keep *_shallowTracks_trackphi_*',
                                      'keep *_shallowTracks_trackhitsvalid_*',
+                                     'keep *_shallowTracks_trackalgo_*',
                                      'keep *_shallowGainCalibration_*_*'
```

Comment:
@dimattia Just to point out that with phase1 this requirement might not work as you would expect (e.g. `highPtTripletStep` that is the triplet version of `initialStep` has a numeric value of 22.
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DataFormats/TrackReco/interface/TrackBase.h#L117